test/Interop/Cxx/stdlib/use-std-iterator.swift

```diff
@@ -0,0 +1,46 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+//
+// REQUIRES: executable_test
+//
+// Enable this everywhere once we have a solution for modularizing libstdc++: rdar://87654514
+// REQUIRES: OS=macosx
+
+import StdlibUnittest
+import StdIterator
+import std.iterator
+import StdVector
+import std.vector
+
+var StdIteratorTestSuite = TestSuite("StdIterator")
+
+StdIteratorTestSuite.test("init") {
+    var vector = Vector()
+    var v1 = Iterator()
+    var _1: CInt = 1
+    vector.push_back(&_1)
+    v1 = begin(vector)
+    expectEqual(v1[0], 1)
+}
+
+StdIteratorTestSuite.test("next") {
+    var v1 = Vector()
+    var _1: CInt = 1, _2: CInt = 2, _3: CInt = 3
+    v1.push_back(&_1)
+    v1.push_back(&_2)
+    v1.push_back(&_3)
+    let n1 = next(begin(v1), 2)
```

Comment:
Can you update this to call `std.next` instead of the next function you created?

Reply:
will do!